schemas/2015-01-01/deploymentTemplate.json

```diff
@@ -201,6 +201,11 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2018-11-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices_containers" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-11-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_blobServices_containers_immutabilityPolicies" },
                   { "$ref": "https://schema.management.azure.com/schemas/2018-11-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts_managementPolicies" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2019-04-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts" },
```

Comment:
>{ "$ref": "https://schema.management.azure.com/schemas/2019-04-01/Microsoft.Storage.json#/resourceDefinitions/storageAccounts" }, [](start = 18, length = 129)

we have a new root schema: 2019-04-01/deploymentTemplate.json. Can you please add them to the new root schema as well?

Reply:
Thanks for the review!
Added.